onnxruntime/test/framework/tunable_op_test.cc

```diff
@@ -0,0 +1,466 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#ifdef _WIN32
+#include <windows.h>
+#endif
+#include <chrono>
+
+#include "core/common/common.h"
+#include "core/framework/tunable.h"
+
+#ifdef _WIN32
+#pragma comment(lib, "WinMM.lib")
+#endif
+
+#ifdef ORT_NO_RTTI
+#define GTEST_SKIP_IF_NO_RTTI() GTEST_SKIP() << "TunableOp needs RTTI to work correctly";
+#else
+#define GTEST_SKIP_IF_NO_RTTI()
+#endif
+
+using namespace std::chrono_literals;
+
+namespace onnxruntime {
+namespace test {
+namespace {
+
+// test on CPU and it does not use stream
+using StreamT = void*;
+
+class Timer : public ::onnxruntime::tunable::Timer<StreamT> {
```

Comment:
Is there value in putting the `void*`/`void` specialization in the tunable op library, instead of in the test code?

Reply:
Nope, since stream is not a usaful concept for CPU code, it is not used.

Reply:
Ideally, we should eliminate the SteamT from `Timer` base, but this will unnecessarily complicate the `FindFastest` impl. If only cuda and rocm will be using it, I think we better leave the refactor for future change. So I think use a dummy void* here for testing is sufficient.